test/e2e/mesos.go

```diff
@@ -87,7 +87,7 @@ var _ = Describe("Mesos", func() {
 			ObjectMeta: api.ObjectMeta{
 				Name: podName,
 				Labels: map[string]string{
-					"k8s.mesosphere.io/roles": "role1",
+					"k8s.mesosphere.io/roles": "public",
 				},
 			},
 			Spec: api.PodSpec{
```

Comment:
This test does not use ports. Let's remove the ports for the public role from the docker-compose.yml. Probably the same for disk.